src/Folder.php

```diff
@@ -109,6 +109,25 @@ public function getIcon()
         );
     }
 
+    /**
+     * Use default File filter rules, but also remove dots and replace with dashes.
+     *
+     * @param string $name
+     * @return string
+     */
+    protected function filterFilename($name)
+    {
+        // Fix illegal characters
+        $filter = FileNameFilter::create();
```

Comment:
Would you be better off with a FileNameFilter subclass (e.g. named service) custom to folders? This is un-injectable.

Reply:
True, fair enough

Reply:
I've done this now =)